src/core/json-schema-components.jsx

```diff
@@ -299,15 +299,15 @@ export class JsonSchemaArrayItemFile extends Component {
   }
 
   render() {
-    let { getComponent, errors, disabled } = this.props
+    let { getComponent, errors, hidden } = this.props
```

Comment:
Why do this change from `disabled` to `hidden` especially as the use remains the same? Changing names makes it harder to sync up future changes.

Reply:
We do not want to show input boxes (even in disabled mode) in "view" mode.
They should appear only when user is trying to make API call to try out the API.

Reply:
@ssat-zededa As per your suggestion, I made changes. Please take a look.